packages/node-runtime/src/http-server.ts

```diff
@@ -74,6 +74,8 @@ export class SdkgenHttpServer<ExtraContextT = unknown> {
     console.log(`${new Date().toISOString()} ${message}`);
   };
 
+  public logError = console.error;
```

Comment:
Especificar a tipagem para não herdar tudo de console.error

```suggestion
  public logError: (error: unknown) => void = console.error;
```